.gitignore

```diff
@@ -1,7 +1,6 @@
 __tests__/runner/*
 
-# comment out in distribution branches
-# node_modules/
+node_modules/
```

Comment:
Do we expect anything to be present in `node_modules/` now?

Reply:
Yes, it still exist during the minification process but we should not commit those files anymore.

Reply:
npm install generates node_modules and ncc generates dist/index.js.